lib/ruby_lsp/server.rb

```diff
@@ -76,6 +76,16 @@ def process_message(message)
         text_document_show_syntax_tree(message)
       when "rubyLsp/workspace/dependencies"
         workspace_dependencies(message)
+      when "rubyLsp/workspace/getAddons"
+        send_message(
```

Comment:
Let's add a test for this.

Reply:
I added a test and renamed it to `rubyLsp/workspace/addons` instead, which is consistent with `dependencies`.